src/parser.ts

```diff
@@ -433,7 +437,7 @@ function variableDefinitions(): ast.VariableDefinitionNode[] | undefined {
         _defaultValue = value(true);
       }
       ignored();
-      vars.push({
+      const varDef: any = {
```

Comment:
This cast seems unnecessary? Could be made safer but since the types have been adapted this should now pass

(same for others here)